ci/requirements/environment.yml

```
@@ -44,6 +44,6 @@ dependencies:
   - sparse
   - toolz
   - typing_extensions
-  - zarr
-  - pip:
-      - numbagg
+  # - zarr  # TODO: Waiting on python 3.11 support.
```

there's a new version of `numcodecs` on `conda-forge` now, so we should be able to undo this:
```suggestion
  - zarr
```